controllers/profiles/reconciler_dev.go

```diff
@@ -51,12 +71,27 @@ const (
 	recoverDeploymentErrorRetries = 3
 	// recoverDeploymentErrorInterval interval between recovering from failures
 	recoverDeploymentErrorInterval = 5 * time.Minute
+	extResCamelSuffix              = "resource-camel"
+	extResGenericSuffix            = "resource-generic"
+	extResOpenAPISuffix            = "resource-openapi"
+	extResAsyncAPISuffix           = "resource-asyncapi"
 )
 
 type developmentProfile struct {
 	baseReconciler
 }
 
+type ConfigPath int
```

Comment:
We could also use an enum type string or at least define a stringer func like this

`func (ConfigPath) String() string`

or use directly the stringer utiliy (https://pkg.go.dev/golang.org/x/tools/cmd/stringer), so it will be easier to print the string corresponding to the enum if it will be needed

Reply:
Ok